### PR TITLE
Add BHNT 92 hacks

### PR DESCRIPTION
**Pull request by ligi**
cc @KaeruCT @wleuschner @weliveindetail @isaksolheim @PiXeL16

### CODE REVIEW

**Comment by isaksolheim** [comment]
@ligi the link to my github profile is incorrect, should be `https://github.com/isaksolheim`

File: _posts/2023-11-28-no92-closedai-hacks.md [hunk 1]
```diff
@@ -9,5 +9,11 @@ image: /assets/images/logo.png
 meetupUrl: https://www.meetup.com/de-DE/berlin-hack-and-tell/events/297499425
 ---
 
-Upcoming - please [RSVP here](https://www.meetup.com/de-DE/berlin-hack-and-tell/events/297499425)
-
+* [my-stop](https://github.com/KaeruCT/my-stop) by [KaeruCT](https://github.com/KaeruCT/my-stop)
+* [PBF2D (Position Based Fluids 2D Tutorial)](https://github.com/wleuschner/pbf_tutorial) by [vlad](https://github.com/wleuschner)
+* 3D printed kensington wrist rest by Patryk
+* [OrcJIT via the LLVM C-API](https://github.com/weliveindetail/llvm-jit-capi) by [weliveindetail](https://github.com/weliveindetail)
+* [wgbot](https://github.com/isaksolheim/gesucht) by [Isak](https://github.com/isaksolheim/gesucht) - **Hack of the Month**
```
Comment (ligi) [inline _posts/2023-11-28-no92-closedai-hacks.md:16]:
```suggestion
* [wgbot](https://github.com/isaksolheim/gesucht) by [Isak](https://github.com/isaksolheim) - **Hack of the Month**
```

File: _posts/2023-11-28-no92-closedai-hacks.md [hunk 2]
```diff
@@ -9,5 +9,11 @@ image: /assets/images/logo.png
 meetupUrl: https://www.meetup.com/de-DE/berlin-hack-and-tell/events/297499425
 ---
 
-Upcoming - please [RSVP here](https://www.meetup.com/de-DE/berlin-hack-and-tell/events/297499425)
-
+* [my-stop](https://github.com/KaeruCT/my-stop) by [KaeruCT](https://github.com/KaeruCT/my-stop)
+* [PBF2D (Position Based Fluids 2D Tutorial)](https://github.com/wleuschner/pbf_tutorial) by [vlad](https://github.com/wleuschner)
+* 3D printed kensington wrist rest by Patryk
+* [OrcJIT via the LLVM C-API](https://github.com/weliveindetail/llvm-jit-capi) by [weliveindetail](https://github.com/weliveindetail)
```
Comment (weliveindetail) [inline _posts/2023-11-28-no92-closedai-hacks.md:15]: That could be more like:
```
[Integer overflow? C → 🤷‍♂️ ⚡️ 😱 ← Rust (panic)](https://github.com/weliveindetail/llvm-jit-capi/commits/rust-panic-hack)
```

**Review by weliveindetail — COMMENTED** [review]
Hey @ligi thanks for organizing the meetup and publishing this!!

File: _posts/2023-11-28-no92-closedai-hacks.md (hunk 2 above)
Comment (ligi) [inline _posts/2023-11-28-no92-closedai-hacks.md:15]:
```suggestion
* [Integer overflow? C → 🤷‍♂️ ⚡️ 😱 ← Rust (panic)](https://github.com/weliveindetail/llvm-jit-capi/commits/rust-panic-hack) by [weliveindetail](https://github.com/weliveindetail)
```

**Review by weliveindetail — APPROVED** [review]
Merci

**Review by lucasrangit — APPROVED** [review]

**Review by PiXeL16 — APPROVED** [review]

**Review by isaksolheim — APPROVED** [review]

**Pull request merged**